internal/proxy/impl.go

```diff
@@ -2306,6 +2306,10 @@ func (node *Proxy) Upsert(ctx context.Context, request *milvuspb.UpsertRequest)
 			errIndex[i] = i
 		}
 
+		if errCode == commonpb.ErrorCode_Success {
+			errCode = commonpb.ErrorCode_UnexpectedError
+		}
+
```

Comment:
how to trigger this condition?

Reply:
has some unexpected error like illegal partition name which will throw error message, but will not change status. It will check the error msg and then change the status to pass the error to sdk.

Reply:
you should handle the error at line 2348, this hacking confuses

Reply:
ok